docs/devel/test-nuclio-locally.md

```diff
@@ -0,0 +1,83 @@
+# Running Nuclio on your computer with local changes
+
+This guide will guide you through the process of setting up Nuclio locally for testing your changes and provide faster development cycles.
```

Comment:
hmm Id title it with "fast development cycles on nuclio services"
and the development here is Nuclio services (dashboard/controller and potentionally dlx/scaler) and not Nuclio's processor (which has different realm and mentioned on the contributing.md)